dist/maptable.js

```diff
@@ -1,4 +1,4 @@
-/* MapTable 1.5.3 */this.d3 = this.d3 || {};
```

Comment:
per my other comment, let's keep this in here so that the dist files have this at the very top. super handy to know which version you're running! since you deleted it here, i'll make this the blocking request.